Configuration/UTMViewState.m

```diff
@@ -78,6 +79,14 @@ - (void)setShowKeyboard:(BOOL)showKeyboard {
     _rootDict[kUTMViewStateShowKeyboardKey] = [NSNumber numberWithBool:showKeyboard];
 }
 
+- (BOOL)suspended {
+    return [_rootDict[kUTMViewStateSuspendedKey] boolValue];
+}
+
+- (void)setSuspended:(BOOL)suspended {
+    _rootDict[kUTMViewStateSuspendedKey] = [NSNumber numberWithBool:suspended];
```

Comment:
`@(suspended)`?

Reply:
I didn't know you can do that.

Reply:
Is there a shorthand for the reverse: `[_rootDict[kUTMViewStateSuspendedKey] boolValue]`?

Reply:
Not that I know of, sadly :(